server/tserver/src/test/java/org/apache/accumulo/tserver/CheckTabletMetadataTest.java

```diff
@@ -66,7 +72,9 @@ private static void assertFail(TreeMap<Key,Value> tabletMeta, KeyExtent ke, TSer
     TreeMap<Key,Value> copy = new TreeMap<>(tabletMeta);
     assertNotNull(copy.remove(keyToDelete));
     try {
-      assertNull(TabletServer.checkTabletMetadata(ke, tsi, copy, ke.getMetadataEntry()));
+      TabletMetadata tm = TabletMetadata.convertRow(copy.entrySet().iterator(),
+          EnumSet.allOf(ColumnType.class), true);
+      assertFalse(TabletServer.checkTabletMetadata(ke, tsi, tm));
     } catch (Exception e) {
 
```

Comment:
This line is now causing a failure in spotbugs, because it is ignoring `Exception`. If the intent is to ignore the exceptions caught, spotbugs can generally be made happy by adding a comment in the block, like:
```suggestion
      // exception is expected
```

I don't know why this didn't bother spotbugs before.